src/frontend/apps/impress/src/components/separators/HorizontalSeparator.tsx

```diff
@@ -30,11 +20,7 @@ export const HorizontalSeparator = ({
       $height="1px"
```

Comment:
```suggestion
      $height="1px"
```
Same as previously if we can have a special var